book/website/reproducible-research/case-studies/statistical-methods-manuscript.md

```diff
@@ -0,0 +1,146 @@
+(rr:cs:statistical-methods-manuscript)=
```

Comment:
In principle, this format for referencing is not wrong. However, if we are trying to standardise these labels to make them findable: https://the-turing-way.netlify.app/community-handbook/style/style-crossref.html
For example: rr-case-studies-statistical-methods-manuscript (this is quite long, but I think it's fine as long as it is readable :))

Reply:
ah, sorry I missed that section in the handbook. My rationale was to distinguish between hierarchy (:) and within-name blanks (-). Consistency trumps that though.